tests/test_postgres.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from test_sqlite import *  # NOQA
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "USER": os.environ["POSTGRES_USER"],
+        "NAME": os.environ["POSTGRES_DB"],
+        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
+        "HOST": os.environ["POSTGRES_HOST"],
+        "PORT": os.environ["POSTGRES_PORT"],
```

Comment:
Is it possible to hard code these so that this file would be useful for both running locally and in CI? 

It would be great if I could run the tests without postgres without having to amend this file, or set environment variables!

Reply:
I would keep it in the `.github/workflows/data/` directory, and copy to `tests/` in one of steps. We could also hard-code values without using environment variables.